src/less/components/map.less

```diff
@@ -25,5 +18,91 @@
       padding: 0 1px;
       width: 10px;
     }
+    &-0 {
+      background-color: @green;
+      border-color: @green-dark;
+      &:hover {
+        background-color: @green-hover;
+      }
+    }
+    &-1 {
+      background-color: @orange;
+      border-color: @orange-dark;
+      &:hover {
+        background-color: @orange-hover;
+      }
+    }
+    &-2 {
+      background-color: @red;
+      border-color: @red-dark;
+      &:hover {
+        background-color: @red-hover;
+      }
+    }
+    &-3 {
+      background-color: @blue;
+      border-color: @blue-dark;
+      &:hover {
+        background-color: @blue-hover;
+      }
+    }
+    &-4 {
+      background-color: @purple;
+      border-color: @purple-dark;
+      &:hover {
+        background-color: @purple-hover;
+      }
+    }
+    &-5 {
+      background-color: @cyan;
+      border-color: @cyan-dark;
+      &:hover {
+        background-color: @cyan-hover;
+      }
+    }
+    &-6 {
+      background-color: @silver;
+      border-color: @silver-dark;
+      &:hover {
+        background-color: @silver-hover;
+      }
+    }
+    &-7 {
+      background-color: @bronze;
+      border-color: @bronze-dark;
+      &:hover {
+        background-color: @bronze-hover;
+      }
+    }
+    &-8 {
+      background-color: @orchid;
+      border-color: @orchid-dark;
+      &:hover {
+        background-color: @orchid-hover;
+      }
+    }
+    &-9 {
+      background-color: @pear;
+      border-color: @pear-dark;
+      &:hover {
+        background-color: @pear-hover;
+      }
+    }
   }
 }
+
+.my-dropdown-slidedown {
+  display: grid;
+}
+
+.slidedown-container {
+  margin-bottom: 10px;
+  span {
```

Comment:
What about adding classes to these elements and not use html tags ? Same for a below.